src/applications/avs/tests/components/YourTreatmentPlan.unit.spec.jsx

```diff
@@ -29,6 +29,11 @@ describe('Avs: Your Treatment Plan', () => {
     expect(screen.getByTestId('other-orders').children[1]).to.contain.text(
       'PACT ALERT BRAVO\nConcern:',
     );
+    expect(
+      screen.getByTestId('your-reminders').children[2].firstChild,

```

Comment:
I'm wondering if we should use `findByTestId` since `get*` methods fire once, while `find*` methods will retry. 

https://depo-platform-documentation.scrollhelp.site/developer-docs/best-practices-for-using-cypress#BestpracticesforusingCypress-Cypresstestinglibraryselectors

Reply:
hmm, this is a mocha unit test rather than a cypress test, do you know if that recommendation still applies?

Reply:
after further discussion in slack we believe this guidance is specific to cypress tests, so resolving this 🙂